### ISSUE TITLE
Cluster GCN demo notebook fails on NetworkX 2.4

### ISSUE THREAD

**Issue by adocherty** [stellargraph/stellargraph#548]
**Describe the bug**
**To Reproduce**
1. Run notebook `demos/node-classification/cluster-gcn/cluster-gcn-node-classification.ipynb`

Execution line 5 gives the following error due to deprecated method `.node`:

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-5-c53f9414d523> in <module>
      1 if dataset == 'cora':
----> 2     Gnx, node_data, feature_names = load_cora(data_dir)
      3 elif dataset == 'pubmed':
      4     Gnx, node_data, feature_names = load_pubmed(data_dir)

<ipython-input-2-52a0b7d59552> in load_cora(data_dir, largest_cc)
     30 
     31     for nid in node_data.index:
---> 32         g_nx.node[nid][globalvar.TYPE_ATTR_NAME] = "paper"  # specify node type
     33 
     34     return g_nx, node_data, feature_names

AttributeError: 'Graph' object has no attribute 'node'
```

**Expected behavior**
No errors for NetworkX 2.4.  This is simply replacing `g_nx.node[` with `g_nx.nodes[`

**Comment by PantelisElinas** [comment]
Hi @adocherty 

was this fixed in a different ticket (I have a vague recollection of this having been fixed), or is it still a problem?

Regards,

P.

**Comment by adocherty** [comment]
This has been fixed in PR #549 for some reason the issue wasn't closed when the PR was closed.
